### DISCUSSION TITLE
Show how long the current rip has been been running

### DISCUSSION THREAD

**Discussion by thompcha** [automatic-ripping-machine/automatic-ripping-machine#684] [Ideas]
I made a userscript that displays the current rip's duration in the duration column of the history tab in the web ui:
https://gist.github.com/thompcha/1a9718512de310ff0c79a0b40a4f7ee1

**Comment by microtechno9000** [comment]
nice work, you are welcome to fork ARM incorporate into the history page and raise a Pull Request.

**Reply by 1337-server** [reply]
This is a userscript.
It's run on the local/browser side of things and saves making changes to the core files themselves.
